addons/project/migrations/11.0.1.1/post-migration.py

```diff
@@ -128,14 +128,14 @@ def convert_issues(env):
                 WHEN pi.priority='2' THEN '1'
                 ELSE pi.priority
             END
-        FROM project_issue pi,
-            project_project pp
-        WHERE pi.project_id = pp.id
+        FROM project_issue pi
+            LEFT JOIN project_project pp ON pp.id = pi.project_id
         """, {
             'project_column': AsIs(issue_project_column),
             'issue_column': AsIs(origin_issue_column),
         },
     )
+
```

Comment:
Why adding extra lines inside a method. That's discouraged by PEP8 and not present in original code.

Reply:
I didn't pay attention to it (my pep8 didn't put a warning  on it), I will remove it